system/gameModes/5e/load.lst

```diff
@@ -1,45 +1,38 @@
-SIZEMULT:T|4
```

Comment:
The 5e gamemode file is obsolete as the pcgen release includes a fixed version. This should just be removed entirely to avoid conflicts.